test/new_tests/as_errors.py

```diff
@@ -127,6 +127,7 @@
 #
 #    A cluster state change occurred during the request. This may also be
 #    returned by scan operations with the fail_on_cluster_change flag set.
+#    TODO: update outdated docstring. fail_on_cluster_change no longer exists
```

Comment:
Is this left here on purpose? Or should it have been done?

Reply:
It's a docstring for an error code in the tests. I didn't want to modify it because I don't know what the new behavior is without the flag